File: Php/tests/joseahernandez/fizzbuzz/FizzBuzzTest.php
<?php

namespace joseahernandez\fizzbuzz;

class FizzBuzzTest extends \PHPUnit_Framework_TestCase
{
    private $fizzBuzz;

    protected function setUp()
    {
        $this->fizzBuzz = new FizzBuzz();
    }


    /**
     * @test
     */
    public function fizzBuzzReturnTheNumberPassedAsArgument()
    {
        $number = 1;
        $result = $this->fizzBuzz->fizzBuzz($number);

        $this->assertSame('1', $result);
    }

    /**
     * @test
     */
    public function multiplesOfThreeReturnFizz() {
        $number = 3;
        $result = $this->fizzBuzz->fizzBuzz($number);

        $this->assertSame('Fizz', $result);
    }

    /**
     * @test
     */
    public function multiplesOfFiveReturnBuzz() {
        $number = 5;
        $result = $this->fizzBuzz->fizzBuzz($number);

        $this->assertSame('Buzz', $result);
    }

    /**
     * @test
     */
    public function multiplesOfThreeAndFiveReturnFizzBuzz() {
        $number = 15;
        $result = $this->fizzBuzz->fizzBuzz($number);

        $this->assertSame('FizzBuzz', $result);
    }
}

File: Swift/FizzBuzzTest/FizzBuzzTest.swift
import XCTest

class FizzBuzzTest: XCTestCase {
    let fizzBuzz = FizzBuzz()
    
    func testFizzBuzzReturnTheNumberPassedAsArgument() {
        let number = 1
        let result = fizzBuzz.fizzBuzz(number)
        
        XCTAssertEqual(String(number), result)
    }
    
    func testMultiplesOfThreeReturnFizz() {
        let number = 3
        let result = fizzBuzz.fizzBuzz(number)
        
        XCTAssertEqual("Fizz", result)
    }
    
    func testMultiplesOfFiveReturnBuz() {
        let number = 5
        let result = fizzBuzz.fizzBuzz(number)
        
        XCTAssertEqual("Buzz", result)
    }
    
    func testMultiplesOfThreeAndFiveReturnFizzBuzz() {
        let number = 15
        let result = fizzBuzz.fizzBuzz(number)
        
        XCTAssertEqual("FizzBuzz", result)
    }
    
}
